src/main/java/org/lk/repository/ApplicationUserAuthorizationRepository.java

```
@@ -0,0 +1,39 @@
+package org.lk.repository;
+
+
+import org.hibernate.Session;
+import org.hibernate.SessionFactory;
+import org.lk.model.domain.UserAuthorizationEntity;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Repository;
+
+import javax.persistence.NoResultException;
+
+@Repository
+public class ApplicationUserAuthorizationRepository {
+
+
+    private final SessionFactory factory;
+
+    @Autowired
+    public ApplicationUserAuthorizationRepository(SessionFactory factory) {
+        this.factory = factory;
+    }
+
+    public UserAuthorizationEntity findById(Integer id) {
+        UserAuthorizationEntity result = null;
+
+        try (Session session = factory.openSession()) {
+
+            result = session.createQuery("from UserAuthorizationEntity where id = :id", UserAuthorizationEntity.class)
+                    .setParameter("id", id)
+                    .getSingleResult();
+        } catch (NoResultException exc) {
+
```

Please, don't use empty catch blocks. You can write a message with a warning or an error and return null instead of creating `result` value at the beginning of the method.